apps/frontend/src/app/components/misc/validation/ErrorField.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactElement } from 'react';
+import style from './ErrorField.module.scss';
+
+export default function ErrorField({errorMsg}: {errorMsg: string}): ReactElement {
```

Comment:
Do we need two components for this functionality? Couldn't the list just render one item if that is what is provided?

Reply:
Agree, i fix